src/accessibility/roving/RovingAccessibleTooltipButton.tsx

```diff
@@ -26,10 +26,12 @@ interface IProps extends Omit<ATBProps, "inputRef" | "tabIndex"> {
 }
 
 // Wrapper to allow use of useRovingTabIndex for simple AccessibleTooltipButtons outside of React Functional Components.
-export const RovingAccessibleTooltipButton: React.FC<IProps> = ({ inputRef, onFocus, ...props }) => {
+export const RovingAccessibleTooltipButton: React.FC<IProps> = ({ inputRef, onFocus, onClick, ...props }) => {
     const [onFocusInternal, isActive, ref] = useRovingTabIndex(inputRef);
     return <AccessibleTooltipButton
         {...props}
+        onClick={onClick}
+        onContextMenu={onClick}
```

Comment:
I don't think this is right, as it now prevents you from having an AccessibleTooltipButton with a different action & context menu, as it overrides whatever onContextMenu was given

Reply:
Good point